thentos-core/src/Thentos/Backend/Core.hs

```diff
@@ -80,13 +81,15 @@ newtype ErrorMessage = ErrorMessage { fromErrorMessage :: ST }
 instance ToJSON ErrorMessage where
     toJSON (ErrorMessage msg) = object ["status" .= String "error", "error" .= msg]
 
+-- |Type of functions that transform a base error and an error message into a ServantErr.
+type MkServantErrFun = ServantErr -> ST -> ServantErr
```

Comment:
I disagree. The type synonym is used not only in the type of `mkServantErr`, but also in the type of `thentosErrorToServantErr` and in `Thentos/Adhocracy3/Types.hs` where an A3-specific MkServantErrFun is defined. Using the type synonym instead of copy-and-pasting the underlying types multiples times makes the code more compact and self-documenting.

It's also advantageous should we ever have reason to change to type of that function -- just check where the type synonym is used to get a grasp of what might need to be changed.